src/public/js/cardModal.js

```diff
@@ -0,0 +1,73 @@
+(function($) {
+    //this file should be imported into the board handlebars file
+    //this variable should be the <div> that the modal is going to be inserted in to
+    let placeModalHere = $("#placeModalHere");
+
+    //this will select every element with an href = populateCard.html;
+    //<a href="/populateCard.html" id="_id of the card">
+    let cards = $("[href=populateCard.html]");
+    for(let x of cards) {
+        let cardId = x.id; //this is why the id needs to be the id of the card
+        let boardId = "TODO"; //I need to get the boardId somehow, since I imagine that's how it works in the routes
+        x.on('click', (e) => {
+            e.preventDefault(); //prevents redirect
+            let requestData = {
+                method: "POST", //I think?
+                url: `TODO/${boardId}/${cardId}` //TODO: Insert route to get info
+            };
+            //make the call to ajax to populate
+            $.ajax(requestData).then(function(responseMessage) {
+                let cardModal = $(responseMessage);
+                placeModalHere.empty(); //handle a modal being there before
+                placeModalHere.append(cardModal); //insert new modal
+                setUpErrorChecking();
```

Comment:
Typically, declare functions above where they are used

Reply:
I'm not sure what you meant by this, could you elaborate?

Reply:
You call setUpErrorChecking() on line 23, but it's not declared until after line 23. In some languages, this wouldn't compile. In general, it's good code practice to move function declarations above their respective function calls

This is no reason not to merge, just mentioning preferred practice